tsconfig.json

```diff
@@ -10,6 +10,11 @@
         }
     },
     "exclude": [
-        "tests/*"
+        "dash_table/*",
+        "dist/*",
+        "node_modules/*",
+        "storybook-static/*",
+        "tests/*",
+        "vv/*"
     ]
```

Comment:
Unrelated, just to help VSCode decide what code it needs to analyze for JS/TS typing, etc.